[2025-02-15T01:37:41.406+0000] {taskinstance.py:1125} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: weather_data_ETL.transform_alerts manual__2025-02-15T00:56:13.638531+00:00 [queued]>
[2025-02-15T01:37:41.422+0000] {taskinstance.py:1125} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: weather_data_ETL.transform_alerts manual__2025-02-15T00:56:13.638531+00:00 [queued]>
[2025-02-15T01:37:41.424+0000] {taskinstance.py:1331} INFO - Starting attempt 5 of 5
[2025-02-15T01:37:41.443+0000] {taskinstance.py:1350} INFO - Executing <Task(PythonOperator): transform_alerts> on 2025-02-15 00:56:13.638531+00:00
[2025-02-15T01:37:41.450+0000] {standard_task_runner.py:57} INFO - Started process 115 to run task
[2025-02-15T01:37:41.455+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'weather_data_ETL', 'transform_alerts', 'manual__2025-02-15T00:56:13.638531+00:00', '--job-id', '77', '--raw', '--subdir', 'DAGS_FOLDER/weather_dag.py', '--cfg-path', '/tmp/tmp31_fl3cd']
[2025-02-15T01:37:41.458+0000] {standard_task_runner.py:85} INFO - Job 77: Subtask transform_alerts
[2025-02-15T01:37:41.574+0000] {task_command.py:410} INFO - Running <TaskInstance: weather_data_ETL.transform_alerts manual__2025-02-15T00:56:13.638531+00:00 [running]> on host 184396a03990
[2025-02-15T01:37:41.823+0000] {taskinstance.py:1568} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airscholar' AIRFLOW_CTX_DAG_ID='weather_data_ETL' AIRFLOW_CTX_TASK_ID='transform_alerts' AIRFLOW_CTX_EXECUTION_DATE='2025-02-15T00:56:13.638531+00:00' AIRFLOW_CTX_TRY_NUMBER='5' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-02-15T00:56:13.638531+00:00'
[2025-02-15T01:37:41.909+0000] {python.py:183} INFO - Done. Returned value was: None
[2025-02-15T01:37:41.937+0000] {taskinstance.py:1368} INFO - Marking task as SUCCESS. dag_id=weather_data_ETL, task_id=transform_alerts, execution_date=20250215T005613, start_date=20250215T013741, end_date=20250215T013741
[2025-02-15T01:37:41.997+0000] {local_task_job_runner.py:232} INFO - Task exited with return code 0
[2025-02-15T01:37:42.048+0000] {taskinstance.py:2674} INFO - 0 downstream tasks scheduled from follow-on schedule check
